pyproject.toml

```diff
@@ -10,62 +10,49 @@ description = "Panpipes - multimodal single cell pipelines"
 readme = "README.md"
 
 classifiers = [
-    # Pick your license as you wish
     "License :: OSI Approved :: BSD License",
-
-    # Optional
-    # How mature is this project? Common values are
-    #   3 - Alpha
-    #   4 - Beta
-    #   5 - Production/Stable
     "Development Status :: 3 - Alpha",
-
-    # Indicate who your project is intended for
     "Intended Audience :: Science/Research",
     "Programming Language :: Python :: 3",
     "Programming Language :: R"
     ]
 
-requires-python = ">= 3.11"
+requires-python = ">= 3.10"
```

Comment:
we can safely do 3.11 imho

Reply:
No, you actually don't want that. The market share of Python 3.11 is very small. This is the MINIMUM required Python version which should also be the lowerbound that your CI is testing for